app/beachbot_controller.py

```diff
@@ -272,13 +314,20 @@ async def grab_video_frame() -> Response:
             ).on_value_change(lambda v: toggle_recoding(v.value == 2))
             with ui.column():
                 with ui.row().classes("w-full"):
-                    do_detect = ui.switch('Blob Detection', on_change=lambda x: toggle_detection(x.value))
+                    with ui.dropdown_button('Select Model', auto_close=True):
+                        ui.item("Detection Off", on_click=lambda: toggle_detection(False, None))
+                        for model in model_list:
+                            ui.item(str(model.__name__), on_click=lambda m=model: toggle_detection(True, m))
                     ui.space()
                     do_control = ui.switch('Robot Control', on_change=lambda x: toggle_control(x.value))
                 
                 with ui.row().classes("w-full justify-between no-wrap"):
-                    ui.label("kp:")
-                    kp_slider = ui.slider(min=0.1, max=100, step=0.1, value=0.1, on_change=lambda x: update_kp(x.value)).props('label')
+                    ui_model_info(robot)
+                    # ui.label("Model:")
+                    # with ui.dropdown_button('Select Model', auto_close=True):
+                    #     ui.item("Detection Off", on_click=lambda: toggle_detection(False, None))
+                    #     for model in model_list:
+                    #         ui.item(str(model), on_click=lambda m=model: toggle_detection(True, m))
```

Comment:
Please remove commented out code from commits. You can keep this in a separate branch if you want it for your own notes. In a branch that won't be merged or used otherwise.